### ISSUE TITLE
windows.info.Info gives an exception

### ISSUE THREAD

**Issue by 2xyo** [volatilityfoundation/volatility3#205]
**Context**

* Volatility Version: Volatility 3 Framework 1.0.0-beta.1 -   83ef338a6e35222ffabffd2c2cf4eb00436e270e 
* Operating System:  Debian GNU/Linux 10 (buster)
* Python Version:  Python 3.7.3 (default, Dec 20 2019, 18:57:59) 
* Suspected Operating System:  Microsoft Windows 10 Enterprise Evaluation - 10.0.18362 N/A Build 18362
  * VM Win10 from https://github.com/cyberdefenders/DetectionLabELK 

**To Reproduce**

1. Use command ': `$ vol -vvvv -f /home/yo/Documents/DetectionLabELK/Vagrant/resultats/WIN10-20200402-154153.bin windows.info.Info`
2. See error: 

```py
Kernel Base 0xf8054fc01000
DTB 0x1aa000
Symbols file:///home/yo/Documents/DFIR/volatility3_git/volatility/symbols/windows/ntkrnlmp.pdb/35A038B1F6E2E8CAF642111E6EC66F57-1.json.xz
primary 0 WindowsIntel32e
memory_layer  1 FileLayerTraceback (most recent call last):
  File "/home/yo/Documents/DFIR/volatility3_venv/bin/vol", line 11, in <module>
    load_entry_point('volatility', 'console_scripts', 'vol')()
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/cli/__init__.py", line 493, in main
    CommandLine().run()
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/cli/__init__.py", line 292, in run
    renderers[args.renderer]().render(constructed.run())
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/cli/text_renderer.py", line 163, in render
    grid.populate(visitor, outfd)
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/renderers/__init__.py", line 196, in populate
    for (level, item) in self._generator:
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/plugins/windows/info.py", line 101, in _generator
    offset = vers_offset)
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/contexts/__init__.py", line 204, in object
    **kwargs)
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/contexts/__init__.py", line 104, in object
    object_template = self._symbol_space.get_type(object_type)
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/symbols/__init__.py", line 170, in get_type
    self._resolved[type_name] = self._weak_resolve(SymbolType.TYPE, type_name)  # type: ignore
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/symbols/__init__.py", line 128, in _weak_resolve
    return getattr(self._dict[table_name], get_function)(component_name)
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/symbols/intermed.py", line 55, in _delegate_function
    return getattr(self._delegate, name)(*args, **kwargs)
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/symbols/intermed.py", line 630, in get_type
    interdict = self._reduce_indirect_members(curdict['fields'][member_name])
  File "/home/yo/Documents/DFIR/volatility3_git/volatility/framework/symbols/intermed.py", line 605, in _reduce_indirect_members
    if members[member].get('anonymous', False):
AttributeError: 'int' object has no attribute 'get'
```

**Expected behavior**

No traceback. Get windows.info.Info info.

**Additional information**
 * full trace https://gist.github.com/2xyo/3ce164e2da3c26166c2c8d618e6d2451#file-trace-txt 
 * imageinfo of the same memory dump with volatility 2.6.1 https://gist.github.com/2xyo/3ce164e2da3c26166c2c8d618e6d2451#file-vol2-txt 
 * RAW memory dump with  `DumpIt 3.0.20200224.1 (X64) (Feb 24 2020)`

**Comment by ikelos** [comment]
Thanks, this should have been resolved by `2cd6a9478ff715b730c790325571a14494d2193e`.  Feel free to reopen this if you still encounter this error after updating...  5:)

**Comment by 2xyo** [comment]
Thanks, issue fixed :)
